prime-router/src/main/kotlin/azure/ApiKeysFunctions.kt

```diff
@@ -174,36 +184,34 @@ SwIDAQAB
 @SecuritySchemes(
     value = [
         SecurityScheme(
-            name = "primeSecurity",
+            name = "primeSecurityOAUTH",
             type = SecuritySchemeType.OAUTH2,
             flows = OAuthFlows(
                 authorizationCode = OAuthFlow(
-                    authorizationUrl = "https://hhs-prime.okta.com/oauth/authorize",
-                    tokenUrl = "https://hhs-prime.okta.com/oauth/token",
+                    authorizationUrl = "https://hhs-prime.oktapreview.com/oauth2/default/v1/authorize",
+                    tokenUrl = "https://hhs-prime.oktapreview.com/oauth2/default/v1/token",
                     scopes = [
                         OAuthScope(
-                            name = "org_admin",
-                            description = "Grants write access to single org"
-                        ),
-                        OAuthScope(name = "prime_admin", description = "Grants access to admin operations"),
-                        OAuthScope(name = "user", description = "Grants read access")
+                            name = "openid",
+                            description = "OpenID Request scope"
+                        )
                     ]
                 )
             ),
             description = "OAUTH2 Authorization for Report Stream API Access."
         ),
-        SecurityScheme(
-            name = "primeSecurityAPIKey",
-            type = SecuritySchemeType.APIKEY,
-            paramName = "x-functions-key",
-            description = "Azure Function Key Authorization for Report Stream API Access."
-        ),
         SecurityScheme(
             name = "primeSecurityServerToServer",
```

Comment:
This worked locally, but I had to use the python script in the repo (or some other means) to get the bearer token. Can we build this into swagger so users can do everything in one place? Server-server is based on OAuth2.0, which swagger supports so I'm thinking it should be possible?

Reply:
chatted in slack convo